packages/console/src/utilities/validator.ts

```diff
@@ -1,6 +1,12 @@
-export const uriValidator = ({ verifyBlank = true }) => {
-  return (value: string) => {
-    if (!verifyBlank && value.trim().length === 0) {
+import { Nullable } from '@silverhand/essentials';
+
+export const uriValidator = ({ allowNullOrEmpty = true }) => {
+  return (value?: Nullable<string>) => {
+    if (!value || value.length === 0) {
+      if (allowNullOrEmpty) {
+        return false;
+      }
+
```

Comment:
Feels unnecessary to make it as a curry function?  just `validate(value: string, options?: { allowEmpty? } )`

Reply:
Also `allowEmpty` should be a separate validate logic, not a `uriValidator` scope.  better keep the methods simple, especially for this validator util methods.

Reply:
Good point!
Updated, PTAL.